tests/file_storage/tests.py

```diff
@@ -85,6 +85,11 @@ def test_deconstruction(self):
 
 class FileStorageTests(SimpleTestCase):
     storage_class = FileSystemStorage
+    # We're not doing "real" database queries, but by changing TIME_ZONE
+    # we may set the database's underlying timezone. It will be set back
+    # when we exit the decorated method, so we don't need a transaction-
+    # aware TestCase type.
+    allow_database_queries = True
```

Comment:
I would switch to `TestCase` instead of this. This attribute was added for backwards compatibility but I don't believe a small performance penalty is worth any risk of leaking some state and having to debug that.